### PR TITLE
Book 2 |  Fig 3.7 Review

### PR DESCRIPTION
**Pull request by VibhutiBansal-11**
### Figure Number
Book 2 Fig 3.7 

### Figures
|Before|After|
|---|---|
|before|After

### Issue 

#39 

## Steps

- [x] Latexified the code
- [x] Jaxified the code
- [x] Added interactive demo for eta and random seed
## Checklist:

- [x] Performed a self-review of the code
- [x] Tested on Google Colab.

## Potential problems/Important remarks

- using font as, font_size = 6 if is_latexify_enabled() else None due to small figure size & overlapping legend

### CODE REVIEW

File: notebooks/book2/03/lkj_numpyro.ipynb [hunk 1]
```diff
@@ -0,0 +1,555 @@
+{
```
Comment (karm-patel) [inline notebooks/book2/03/lkj_numpyro.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # We illustrate LKJ prior as discussed in fig 14.3 of
</pre><p>You can use <code>%capture</code> to hide this installation logs</p><p>Dr. Kevin won't like this extra logs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by karm-patel** [comment]
Nice improvemen! After resolving this one comment, you can go ahead.

File: notebooks/book2/03/lkj_numpyro.ipynb (hunk 1 above)
Comment (nipunbatra) [inline notebooks/book2/03/lkj_numpyro.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #10.        import numpyro.distributions as dist
</pre><p>We should use distrax or tfp.distributions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (nipunbatra) [inline notebooks/book2/03/lkj_numpyro.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #17.    import numpy as np
</pre><p>I think we are not using numpy and should remove it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (nipunbatra) [inline notebooks/book2/03/lkj_numpyro.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #22.    np.set_printoptions(precision=3)
</pre><p>suggested to remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (nipunbatra) [inline notebooks/book2/03/lkj_numpyro.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #7.        R = dist.LKJ(dimension=2, concentration=eta).sample(jax.random.PRNGKey(0), (int(1e4),))
</pre><p>It may be good to add a comment on what the second parameter is and why you have kept it to 1e4</p><p>Can we choose a more informative variable name rather than R</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by VibhutiBansal-11** [comment]
@karm-patel , I have made all the changes suggested by sir. I have renamed the file from lkj_numpyro to lkj_1d because we are not using numpyro library anymore.
